<!-- -*- mode: XML -*- args="1" -->
<launch>

<arg name="laser_vangle" value="
  -20.5, -19.5, -18.5, -17.5,
  -16.5, -15.5, -14.5, -13.5,
  -12.5, -11.5, -10.5,  -9.5,
   -8.5,  -7.5,  -6.5,  -5.5,
   -4.5,  -3.5,  -2.5,  -1.5,
   -0.5,   0.5,   1.5,   2.5,
    3.5,   4.5,   5.5,   6.5,
    7.5,   8.5,   9.5,   10.5,  "/>

<arg name="laser_hangle" value="
  0,  0,  0,  0,
  0,  0,  0,  0,
  0,  0,  0,  0,
  0,  0,  0,  0,
  0,  0,  0,  0,
  0,  0,  0,  0,
  0,  0,  0,  0,
  0,  0,  0,  0,
  0,  0,  0,  0,  "/>

<!-- device 1 : ip_192.168.0.3  port_2014 -->
<node pkg="rfans_driver" type="driver_node" name="rfans_driver" >
  <param name="advertise_name" value="rfans_packets" />
  <param name="control_name" value="rfans_control"/>

  <param name="device_ip" value="192.168.0.3" />
  <param name="device_port" value="2014" />
  <param name="rps" value="10"/>
  <param name="frame_id" value="/world"/>

</node>

<node pkg="rfans_driver" type="calculation_node" name="calculation_node" >
  <param name="advertise_name" value="rfans_points" />
  <param name="subscribe_name" value="rfans_packets" />
  <param name="angle_duration" value="360" />
  <param name="laser_vangle" value="$(arg laser_vangle)"/>
  <param name="laser_hangle" value="$(arg laser_hangle)"/>
</node>

<!-- device 2 : ip_192.168.0.5  port_3014  -->
<!--
<node pkg="rfans_driver" type="driver_node" name="rfans_driver_05" >
  <param name="advertise_name" value="rfans_packets_05" />
  <param name="device_ip" value="192.168.0.5" />
  <param name="device_port" value="2019" />
  <param name="rps" value="10"/>
</node>

<node pkg="rfans_driver" type="calculation_node" name="calculation_node_05" >
  <param name="advertise_name" value="rfans_points_05" />
  <param name="subscribe_name" value="rfans_packets_05" />
  <param name="angle_duration" value="180" />
</node>-->


</launch>
